src/community/ogcapi/ogcapi-core/src/main/java/org/geoserver/api/APIFilterParser.java

```diff
@@ -17,35 +19,53 @@
 
     private static final FilterFactory2 FF = CommonFactoryFinder.getFilterFactory2();
     public static String CQL_TEXT = "cql-text";
+    public static String CQL_OBJECT = "cql-object";
```

Comment:
According to spec, should be "cql-json":

https://github.com/opengeospatial/ogcapi-features/blob/master/extensions/cql/standard/clause_6_filter.adoc#parameter-filter-lang

Looks like

Reply:
fixed